apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -51,11 +51,16 @@ const blankNewStudentRow = {
 };
 
 // Initial state for the manageStudents redux store.
+
+// addStatus should be an object with the following keys
+// status - AddStatus enum describing the result of the add
+// numStudents - number of students that were attempted to add
```

Comment:
1. Why doesn't it have these keys in its initial state?
2. This is a thing Immutable.js is really good at :grin: but not sure it's worth pivoting right now.